user/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls  import path
+
+from .            import views
+
+app_name = 'user'
+urlpatterns = [
+        path('signup/', views.SignUpView.as_view(), name = 'signup'),
+        path('signin/', views.SignInView.as_view(), name = 'signin'),
```

Comment:
```suggestion
        path('/signup', views.SignUpView.as_view(), name = 'signup'),
        path('/signin', views.SignInView.as_view(), name = 'signin'),
```
앱 url의 앞에 슬래시를 붙이는 것이 위코드 컨벤션입니다 :)

Reply:
네 수정하였습니다!